Makefile

```diff
@@ -453,6 +477,9 @@ generate-metrics-config: $(ENVSUBST_BIN) ## Generate ./hack/observability/kube-s
 		fi \
 	done >> "$${OUTPUT_FILE}"; \
 
+.PHONY: generate-e2e-templates
```

Comment:
Can we move this down to the templates section?



I wonder if we should prefix all those new generate targets with test- . The existing generate targets are all to generate something for our regular providers and code/YAML that is checked-in. Those generate targets are all just for e2e tests, so it might be confusing.

@fabriziopandini WDYT?

Reply:
Thinking a bit more. I'm fine with keeping the name. But let's move this down to templates